### PR TITLE
Update journal-week-2.md

### PR DESCRIPTION
**Pull request by jmcreasman**
Journal, Week 2
 Rough that draft!
 :+1: x2 for me...
 :+1: x2 for you... (with links!)
 Park those extra ideas for later...

### CODE REVIEW

**Comment by nickycadavillo** [comment]
:+1:  Congrats on completing all the assignments this week! It has surely been a rough two weeks for me as well. I like that this has solidified your choice of becoming a developer - keep swimming!  I suggest adding more technical notes/definitions on what we learned to keep things fresh in your memory. Draft looks awesome!!!!!

**Comment by ghost** [comment]
:+1:  You voiced a lot of concerns I've had, too. The whole studying vs. filling out these damn boxes can be a delicate balance.  IMHO, you're doing a great job. Keep being a role model! :)

File: journal-week-2.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
...
+
+This weekend was really a turning point for me as I got to experience how this immersion teaching style is effective. And the understanding that I am not going to comprehend 100% of what is taught right away, but that I should persevere knowing that it comes in time. I also really enjoyed the weekend’s assignment as I got to see how everything we had been learning would come together as a Front End Developer. By putting together the css, html and javascript to create web pages I knew that this was a career I wanted to pursue.
+
+I have also already began thinking about what kind of job I’d like to have in the future after I graduate from the Iron Yard. I come from a marketing background and my previous job before coming here was at a small marketing agency doing web design. I LOVED what I did there but could not really take my job any further due to my lack of knowledge in coding. That was a huge reason why I made the decision to come study here at the Iron Yard. I would like to go back to a marketing agency and apply what I have learned here along with my knowledge of photoshop and copy writing to create/design websites.
```
Comment (kellymurray) [inline journal-week-2.md:6]: Yay!

**Comment by Ricecoder** [comment]
Great insight into the whole doing the assignment/taking the time to learn the concepts. It's so easy to push aside the learning part in favor of doing but it is the learning part the really helps. Also it's great that you're thinking ahead to jobs and what's next after TIY. :+1:

**Comment by jeffworley** [comment]
Congrats on the mentality shift Jon!  In my personal experience not only with coding but with life, deciding to make understanding a focus reaps nothing but benefits.  In an effort to help you possibly revise for the second round of judgements, you may want to add a quick 3 sentences on the big topics other than DOM we covered last week and use it as a timeline rather than as a resource like our CheatSheets. :+1:

**Comment by cmauban** [comment]
Glad to hear that this past weekend was a turning point for you! I am too now understanding the effectiveness of an immersive teaching style. One thing I would incorporate in your journal entry is one of the topics we were given. Connect what we learned to one of the ideas or MVC. :+1:

**Pull request merged**